internal/consensus/state.go

```diff
@@ -1751,10 +1752,8 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 	cs.metrics.MissingValidatorsPower.Set(float64(missingValidatorsPower))
 
 	// NOTE: byzantine validators power and count is only for consensus evidence i.e. duplicate vote
-	var (
-		byzantineValidatorsPower = int64(0)
-		byzantineValidatorsCount = int64(0)
-	)
+	var byzantineValidatorsPower, byzantineValidatorsCount int64
```

Comment:
nit: I'd keep these on two lines to maintain readability.

Reply:
updated, thanks for reminding